openmrs/src/test/java/org/motechproject/openmrs/resource/impl/ObservationResourceImplTest.java

```diff
@@ -78,6 +79,23 @@ public void shouldQueryForObservationByPatientId() throws Exception {
     }
 
     @Test
+    public void shouldQueryForObservationByPatientIdAndConceptId() throws Exception {
+        String patientId = "OOO";
+        String conceptId = "CCC";
+        URI url = config.toInstancePathWithParams("/obs?patient={patientUUID}&concept={conceptUUID}&limit=1", patientId, conceptId);
+
+        when(restOperations.exchange(eq(url), eq(HttpMethod.GET), any(HttpEntity.class), eq(String.class)))
+                .thenReturn(getResponseFromFile(OBSERVATION_QUERY_RESPONSE_JSON));
+
+        ObservationListResult result = observationResource.getObservationByPatientUUIDAndConceptUUID(config, patientId, conceptId);
+
+        verify(restOperations).exchange(eq(url), eq(HttpMethod.GET), requestCaptor.capture(), eq(String.class));
+
+        assertThat(result, equalTo(readFromFile(OBSERVATION_QUERY_RESPONSE_JSON, ObservationListResult.class)));
+        assertThat(requestCaptor.getValue().getHeaders(), equalTo(getHeadersForGet(config)));
+        assertThat(requestCaptor.getValue().getBody(), nullValue());
```

Comment:
please remove this assert, I know that other tests have this line, however this check doesnt have much sense